spec/system/candidate_interface/continuous_applications/viewing-application-choices/candidate_views_personal_statement_spec.rb

```diff
@@ -0,0 +1,108 @@
+require 'rails_helper'
+
+RSpec.describe 'Personal statement', :js do
+  include CandidateHelper
+
+  before do
+    create_and_sign_in_candidate
+    @application_form = create(:application_form, candidate: current_candidate)
+  end
+
+  scenario 'when application is unsubmitted and showing full personal statement' do
+    given_i_have_an_unsubmitted_application_with_short_personal_statement
+    when_i_visit_my_applications
+    when_i_click_to_view_my_application
+    then_i_should_see_the_full_personal_statement
+  end
+
+  scenario 'when application is unsubmitted and personal statement is long' do
+    given_i_have_an_unsubmitted_application_with_long_personal_statement
+    when_i_visit_my_applications
+    when_i_click_to_view_my_application
+    save_and_open_page
+    then_i_should_see_only_the_short_personal_statement
+    when_i_click_show_more
+    then_i_should_see_the_whole_personal_statement
+    when_i_click_show_less
+    then_i_should_see_only_the_short_personal_statement
+  end
```

Comment:
Humm this test is failing because of the `govuk-visually-hidden`. Is hidden to the user but capybara verifies the element content is there on the page 🤔